dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/gist/DefaultGistAccessControl.java

```diff
@@ -90,7 +90,13 @@ public String getCurrentUserUid()
     @Override
     public boolean isSuperuser()
     {
-        return currentUser != null && !currentUser.isSuper();
```

Comment:
Whoops! `!currentUser.isSuper()` => `currentUser.isSuper()` 

Tests did not find this because so far this only had an effect on the sharing filter which in unit tests is skipped since the in memory database does not support the JSONB functions.